meson.build

```diff
@@ -1669,6 +1669,15 @@ if conf.get('ENABLE_LOGIND') == 1
         endif
 endif
 
+# user@.service runs even if logind is not enabled, so this generator should too
```

Comment:
Yeah, I think it's best to drop this part.

Then the question arises how to do the conversion. Just doing a simple `grep UserTasksMax= /etc/systemd/logind.conf` will most likely cover 90% of cases. To do a comprehensive conversion, it'd be much easier to do if there was an easy way to gather all the configuration snippets. Something like `systemctl cat`, but for configuration files. What do you think about this approach? As this generator shows, it'd be mostly a matter of calling `config_parse_many_nulstr` will the right list of directories. This could live in all the `journalctl`/`networkctl`/`loginctl` helpers, but I think it's a niche use case.  I think a single verb in `systemd-analyze` would be best.

Reply:
do we really need to convert? i mean, i'd just continue to process it the way we currently do, i.e. it would continue to work as well or as badly as it currently does, which means it's not a regression if it doesn't work for sessions started outside of logind, but I think it's totally fine if that's the case, no?

That said, it might be good to have a generic systemd-analyze verb that allows coalescing configuration files following out logic. Should be generic though, i.e. not special cases for logind or so, but so that it's easy to put together the right cmdline for any of the files we read, including tmpfiles.d or modules-load.d or anything else...